include/openenclave/enclave.h

```diff
@@ -724,6 +724,76 @@ oe_enclave_t* oe_get_enclave(void);
  */
 oe_result_t oe_random(void* data, size_t size);
 
+/**
+ * oe_generate_attestation_cert.
```

Comment:
Don't use abbreviations ("cert") in non-POSIX API names.

Reply:
thanks for spotting this. 
Renamed to 
oe_generate_attestation_certificate
oe_free_attestation_certificate
oe_verify_attestation_certificate